ClusterKit/Core/CKCluster.h

```diff
@@ -65,7 +65,7 @@ MK_EXTERN NSComparisonResult MKMapSizeCompare(MKMapSize size1, MKMapSize size2);
 /**
  Cluster coordinate.
  */
-@property (nonatomic, readonly) CLLocationCoordinate2D coordinate;
+@property (nonatomic) CLLocationCoordinate2D coordinate;
```

Comment:
I don't think this change is required as `CKCluster` implements the `MKAnnotation` method [`setCoordinate:`](https://developer.apple.com/documentation/mapkit/mkannotation/1429528-setcoordinate).